assets/src/edit-story/components/popup/index.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { createPortal } from 'react-dom';
+import styled from 'styled-components';
+import { useLayoutEffect, useState } from 'react';
+
+const DEFAULT_WIDTH = 270;
+const MAX_HEIGHT = 370;
+
+const Container = styled.div`
+  position: fixed;
+  z-index: 2147483646;
+  top: ${({ y }) => `${y}px`};
+  right: ${({ x }) => `${x}px`};
```

Comment:
As these values are continuously changing on scroll, they should be set using `style` rather than CSS class. Just moving them to an `.attrs()` above, should work:

```js
const Container = styled.div.attrs((x,y}) => ({
  style: { right: `${x}px`, top: `${y}px` },
}))`
```